doc/build-unix.md

```diff
@@ -117,6 +117,12 @@ To build with Qt 5 (recommended) you need the following:
 
     sudo apt-get install libqt5gui5 libqt5core5a libqt5dbus5 qttools5-dev qttools5-dev-tools libprotobuf-dev protobuf-compiler
 
+If ```./configure``` fails to detect your QT installation, you have to run:
+
+		sudo apt-get install libqt5*
+
+to install all QT5 packages available for your Linux Distribution.
```

Comment:
This does not hold true for all Linux distributions.Is this maybe only a problem on debian? If yes, I am sure there is a single at package that can pass the configure check.

Reply:
not to mention that passing unescaped star wildcard is dangerous

Reply:
@TheCharlatan @tomasbrod  Sadly I cant check it now afterwards, because I installed the packages as i described in the commit. To remove them all would mean to uninstall some Applications like VLC and Telegram and the work that has to be done would be too big to justify a change. We can maybe mark the procedure as a last measure if Automake wont detect QT at al.